micro-benchmarks/DRB160-gpu-missingtaskdependency-orig-yes.c

```diff
@@ -0,0 +1,58 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+*/
+
+/*
+Missing dependencies between the addition@39 and multiplication@45 task.
+Data Race Pairs: x[i]@39:11 and x[i]@39:22; x[i]@45:11 and x[i]@45:18.
```

Comment:
@xintin Can the code be simplified further so that there is only one pair of source locations cause data races?

Reply:
Yes @chunhualiao. I will make the necessary changes and will resubmit the request.